Mage.Sets/src/mage/cards/w/WailOfWar.java

```diff
@@ -0,0 +1,52 @@
+package mage.cards.w;
+
+import mage.abilities.Mode;
+import mage.abilities.effects.common.ReturnFromGraveyardToHandTargetEffect;
+import mage.abilities.effects.common.continuous.BoostControlledEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Duration;
+import mage.filter.StaticFilters;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.target.TargetPlayer;
+import mage.target.common.TargetCardInGraveyard;
+
+import java.util.UUID;
+
+/**
+ * Represents the "Wail of War" instant card.
+ * Author: @mikejcunn
+ */
+public final class WailOfWar extends CardImpl {
+
+    public WailOfWar(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.INSTANT}, "{2}{B}");
+
+        // Mode 1: Debuff opponent's creatures
+        Mode debuffOpponentCreaturesMode = new Mode(new BoostControlledEffect(
```

Comment:
you don't need to create a mode for the first mode, just add the effects and targets as normal